cves/kernel/CVE-2017-17806.yml

```diff
@@ -474,9 +512,7 @@ CWE_instructions: |
     CWE: 123            # also ok
 CWE:
 - 787
-CWE_note: |
-  CWE as registered in the NVD. If you are curating, check that this
-  is correct and replace this comment with "Manually confirmed".
+CWE_note: "Manually confirmed"
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
```

Comment:
Question for both of these: what happened to the big list of class-related things/defense strategies?